packages/1155-contracts/src/delegation/ZoraCreator1155PremintExecutorImpl.sol

```diff
@@ -43,6 +43,12 @@ interface ILegacyZoraCreator1155PremintExecutor {
     ) external payable returns (uint256 newTokenId);
 }
 
+struct MintArguments {
```

Comment:
Natspec here for where these get passed?

MintRecipient and MintComment go to fixedpriceminter, while mintReferral is used in the 1155impl contract.